internal/api/controllers/public/version.go

```diff
@@ -0,0 +1,15 @@
+package public
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+type Response struct {
```

Comment:
No need to define the model. If you define the response schema in the schema file and re-generate the models (`make generate-api`) the model will be created for you. See https://github.com/RedHatInsights/playbook-dispatcher/blob/master/internal/api/controllers/private/types.gen.go#L27 for an example.

Reply:
Oh, thank you! That makes more sense.

I tried running `make generate-api `in the master branch, but I'm getting an error
```
make generate-api 
# public API
~/go/bin/oapi-codegen -generate server,spec -package public -o internal/api/controllers/public/spec.gen.go schema/public.openapi.yaml
~/go/bin/oapi-codegen -generate types -package public -o internal/api/controllers/public/types.gen.go schema/public.openapi.yaml
# internal API
~/go/bin/oapi-codegen -generate server,spec -package private -o internal/api/controllers/private/spec.gen.go -import-mapping=./public.openapi.yaml:playbook-dispatcher/internal/api/controllers/public schema/private.openapi.yaml
~/go/bin/oapi-codegen -generate types -package private -o internal/api/controllers/private/types.gen.go -import-mapping=./public.openapi.yaml:playbook-dispatcher/internal/api/controllers/public schema/private.openapi.yaml
[comitche@2603-6080-1540-1f67-fefb-8bf4-2d13-806b playbook-dispatcher]$ make test
ACG_CONFIG=/home/comitche/Projects/test/playbook-dispatcher/cdappconfig.json go run . migrate up
# playbook-dispatcher/internal/api/controllers/public
internal/api/controllers/public/spec.gen.go:240:29: undefined: openapi3.T
internal/api/controllers/public/spec.gen.go:243:12: undefined: openapi3.NewLoader
internal/api/controllers/public/spec.gen.go:245:40: undefined: openapi3.Loader
internal/api/controllers/public/types.gen.go:51:6: CreatedAt redeclared in this block
        previous declaration at internal/api/controllers/public/types.gen.go:40:25
make: *** [Makefile:49: migrate-db] Error 2
```

Reply:
It looks like this is being generated and causing an issue in `internal/api/controllers/public`

```
...
// Defines values for RunsSortBy.
const (
	CreatedAt RunsSortBy = "created_at"

	CreatedAtAsc RunsSortBy = "created_at:asc"

	CreatedAtDesc RunsSortBy = "created_at:desc"
)

// Identifier of the tenant
type Account string

// A timestamp when the entry was created
type CreatedAt time.Time
...
```

Reply:
At the top of the file, this line is generated:
```
// Package public provides primitives to interact with the openapi HTTP API.
//
// Code generated by github.com/deepmap/oapi-codegen version v1.9.1 DO NOT EDIT.
```
It is showing `github.com/deepmap/oapi-codegen v1.4.2` in the go.mod file, but v1.9.1 here.

Reply:
Likely. How did you install oapi-codegen? Did you do that by running `make init`?

Reply:
I think I ran `go get ./...`. I reinstalled the modules with make init and now it's working properly!